### PR TITLE
SONAR-6384 add java WS to cancel all plugins install/update/uninstall

### PR DESCRIPTION
**Pull request by sns-seb**
adds new WS in Java to cancel all updates/installs of plugins (ie. download) and uninstalls
new WS is bound to URL /api/plugins/cancel_all

### CODE REVIEW

**Comment by simonbrandhof** [comment]
LGTM

File: server/sonar-server/src/main/java/org/sonar/server/plugins/ws/CancelAllPluginsWsAction.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  @Override
+  public void define(WebService.NewController controller) {
+    controller.createAction("cancel_all")
+        .setPost(true)
+        .setDescription("Cancels any operation pending on any plugin (install, update or uninstall)")
+        .setHandler(this);
+  }
+
+  @Override
+  public void handle(Request request, Response response) throws Exception {
```
Comment (simonbrandhof) [inline server/sonar-server/src/main/java/org/sonar/server/plugins/ws/CancelAllPluginsWsAction.java]: Security - check that user is a system administrator.

**Comment by simonbrandhof** [comment]
Removed my previous LGTM comment - see blocker comment about permissionning.

**Comment by sns-seb** [comment]
PR updated to take blocker comment into account and rebase on master

**Comment by simonbrandhof** [comment]
Added comment about description, then LGTM

**Pull request merged**